lib/Path/Tiny.pm

```diff
@@ -115,11 +115,15 @@ sub _symbolic_chmod {
 # or detect that, we warn once instead of being fatal if we can detect it and
 # people who need it strict can fatalize the 'flock' category
 
+# Some file systems (eg lustre) may throw up 'function not implemented' on flock.
+# Again, we warn once instead of making this fatal.
+
 #<<< No perltidy
-{ package flock; use if Path::Tiny::IS_BSD(), 'warnings::register' }
+{ package flock; use if 1, 'warnings::register' }
 #>>>
 
 my $WARNED_BSD_NFS = 0;
+my $WARNED_FUNCTION_NOT_IMPL = 0;
```

Comment:
I would call this `$WARNED_FLOCK_NOT_IMPL` since it’s a global but it’s about `flock` specifically rather than just any kind of unimplemented function.